src/style.css

```diff
@@ -2,0 +1,70 @@
+body {
+  margin-right: 8px;
+  margin-left: 8px;
+  font-family: Avenir, sans-serif;
```

Comment:
The `font-family` property should have a fallback font in case the main one doesn't load. For example, you could write `font-family: Avenir, Arial, sans-serif;`.